samples/highcharts/demo/histogram/demo.html

```diff
@@ -8,7 +8,9 @@
     <div id="container"></div>
     <p class="highcharts-description">
         Chart showing how Highcharts can automatically compute a histogram from
-        source data. In this chart, the source data is also displayed as a
-        scatter plot.
+        data. Histograms display count data for one-dimensional datasets, with
+        the x-axis denoting the buckets, and the y-axis denoting the count
+        within a bucket. The scatterplot aids in visualizing the underlying
```

Comment:
In the Highcharts API, we use the term `bin` for bucket. See for example [binWidth](https://api.highcharts.com/highcharts/series.histogram.binsNumber). Maybe we should use that in the demo too?